src/multi/bmp180_internal.hpp

```diff
@@ -6,67 +6,83 @@
 
 namespace hal::sensor::bmp180_internal {
 
-    // Struct to hold the computation variables. The names of the computation
-    // variables are taken directly from the datasheet. The datasheet does not
-    // explain what these names mean, so we keep the names as they are defined.
-    struct computation_variables
-    {
-    std::int32_t x1, x2, x3;
-    std::int32_t b3, b5, b6;
-    std::uint32_t b4, b7;
-    std::int32_t temperature, pressure;
-    };
+// Struct to hold the computation variables. The names of the computation
+// variables are taken directly from the datasheet. The datasheet does not
+// explain what these names mean, so we keep the names as they are defined.
+struct computation_variables
+{
+  std::int32_t x1, x2, x3;
+  std::int32_t b3, b5, b6;
+  std::uint32_t b4, b7;
+  std::int32_t temperature, pressure;
+};
 
-    // Compute the temperature in steps of 0.1C.
-    // These calculations are done the way they are without bit_modify as its
-    // following the exact algorithm for conversion given in figure 4 of the
-    // datasheet.
-    inline computation_variables calculate_true_temperature(std::int32_t p_uncompensated_temperature, std::uint16_t p_ac5, std::uint16_t p_ac6, std::int16_t p_mc, std::int16_t p_md) {
-        computation_variables variables;
+// Compute the temperature in steps of 0.1C.
+// These calculations are done the way they are without bit_modify as its
+// following the exact algorithm for conversion given in figure 4 of the
+// datasheet.
+inline computation_variables calculate_true_temperature(
```

Comment:
You could actually set this to be contexpr since there is nothing inside that cannot be computed at compile time.

Reply:
I'll just resolve this now actually.